src/software/simulated_tests/simulated_test_fixture.py

```diff
@@ -0,0 +1,221 @@
+import threading
+import argparse
+import time
+import os
+
+import pytest
+import software.geom.geometry as tbots_geom
+from proto.geometry_pb2 import Angle, AngularVelocity, Point, Vector
```

Comment:
can we make a python module that does all these imports so we can just import one thing, like `from tbots_proto import *`: https://stackoverflow.com/questions/46688216/in-python3-does-import-work-transitively

Reply:
we'd have to manually maintain it if we just added the imports there

added a script called `import_all_protos` to the proto folder that dynamically loads all proto classes and replaced where we were practically using all the protobufs